sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1494,7 +1494,7 @@ object SQLConf {
       " register class names for which data source V2 write paths are disabled. Writes from these" +
       " sources will fall back to the V1 sources.")
     .stringConf
-    .createWithDefault("csv,json,orc,text")
+    .createWithDefault("csv,json,orc,text,parquet")
```

Comment:
Just an idea: what about using aliases (name) of the respective providers? That would make the list more type-safe and easily searchable.

Reply:
The current way is simpler. We also use the short name in `df.write.format(format)` or `df.read.format(format)`.